### ISSUE TITLE
problem with training

### ISSUE THREAD

**Issue by kimchicanh** [dbolya/yolact#316]
Hi,
I use python3.6
I train model with my dataset (format: coco) and this is log of training:
Warning: Moving average ignored a value of nan
Warning: Moving average ignored a value of nan
Warning: Moving average ignored a value of inf
[  0]       0 || B: 0.000 | C: 0.000 | M: 0.000 | S: 2.836 | T: 2.836 || ETA: 23 days, 21:11:17 || timer: 1.290
Warning: Moving average ignored a value of nan
Warning: Moving average ignored a value of nan
Warning: Moving average ignored a value of inf
Warning: Moving average ignored a value of nan
Warning: Moving average ignored a value of nan
Warning: Moving average ignored a value of inf
Warning: Moving average ignored a value of nan
Warning: Moving average ignored a value of nan
Warning: Moving average ignored a value of inf

Thanks

**Comment by dbolya** [comment]
Hmm, are you using the most up-to-date version of the repo? There was an issue with exploding loss that's been fixed.

If you are on the latest version of the repo, could it be that the model is instantly overfitting? You're getting 0 loss for box, class, and masks. How many images / annotations do you have in your dataset?

**Comment by ShawnNew** [comment]
I also get this kind of error while training, the difference is that all the losses become very large number and then the program stopped. I am indeed using the latest repo.
